src/lib/components/schedule/schedule-frequency.svelte

```diff
@@ -13,7 +14,7 @@
 <div class="flex flex-col {$$props.class}">
   {#if calendar}
     <p data-testid="schedule-calendar-frequency">
-      {calendar?.comment ?? ''} UTC
+      {structuredCalendarToFrequency(calendar) || calendar?.comment || ''}
```

Comment:
Could we just move the `|| calendar?.comment || ''` logic to `structuredCalendarToFrequency` ?

Reply:
good call